web_src/js/features/repo-common.ts

```diff
@@ -53,20 +53,34 @@ export function substituteRepoOpenWithUrl(tmpl: string, url: string): string {
 function initCloneSchemeUrlSelection(parent: Element) {
   const elCloneUrlInput = parent.querySelector<HTMLInputElement>('.repo-clone-url');
 
-  const tabSsh = parent.querySelector('.repo-clone-ssh');
-  const tabHttps = parent.querySelector('.repo-clone-https');
+  const tabHTTPS = parent.querySelector('.repo-clone-https');
+  const tabSSH = parent.querySelector('.repo-clone-ssh');
+  const tabTea = parent.querySelector('.repo-clone-tea');
   const updateClonePanelUi = function() {
     const scheme = localStorage.getItem('repo-clone-protocol') || 'https';
-    const isSSH = scheme === 'ssh' && Boolean(tabSsh) || scheme !== 'ssh' && !tabHttps;
-    if (tabHttps) {
-      tabHttps.textContent = window.origin.split(':')[0].toUpperCase(); // show "HTTP" or "HTTPS"
-      tabHttps.classList.toggle('active', !isSSH);
+    const isHTTPS = scheme === 'https' && Boolean(tabHTTPS) || scheme !== 'https' && !tabHTTPS;
+    const isSSH = scheme === 'ssh' && Boolean(tabSSH) || scheme !== 'ssh' && !tabSSH;
+    const isTea = scheme === 'tea' && Boolean(tabTea) || scheme !== 'tea' && !tabTea;
```

Comment:
Why check `!tabTea` here? It always exists (you added it without `if` check)

And need to confirm again: does `tea` CLI work with only "https" or only "ssh"? Or both?

Reply:
The tab doesn't always exists since choices are also here:

<img width="623" alt="image" src="https://github.com/user-attachments/assets/bbc65ae6-254a-4624-874e-734971769819" />

But since GitHub doesn't add their CLI command here, I haven't done it there too (+ it needs more work to change the pre-made commands below with the tea command)

---

And for now the tea cli command is always shown (again like GitHub), but I can change that if needed.